tests/mul.s

```diff
@@ -0,0 +1,8 @@
+mul $3 $2 #1
+print #1
+mul #1 $50 #2
```

Comment:
I think this case should change to test `neg` * `pos`, since the `MUL` operand may acted different, your test case should iterate all possible combinations.